pymc4/random_variables/types.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
should this live in `random_variables`?

Reply:
That could work too instead.

Reply:
Sorry, I actually like that it's a separate file, but I don't see why it should be in the subdir.